code/drasil-printers/lib/Language/Drasil/Markdown/CreateMd.hs

```diff
@@ -97,16 +97,21 @@ license :: Doc -> Doc
 license auth = text "Copyright (c) 2021," <+> auth <>
   text ". All rights reserved. Please see the [full license](https://github.com/JacquesCarette/Drasil/blob/4b9ad0a3016fecb3c7a2aa82ab142f9e805b5cc8/LICENSE) for more details."
 -- | Drasil Tree icon. Uses HTML directly to format image since normal markdown doesn't support it.
-drasilImage :: Doc
-drasilImage = alignImage "../../../../drasil-website/WebInfo/images/Icon.png"
+drasilImage :: String -> Doc
+drasilImage name = if name == "Projectile" then 
+                      alignImage 
+                        "../../../../../drasil-website/WebInfo/images/Icon.png"
+                   else
+                      alignImage 
+                        "../../../../drasil-website/WebInfo/images/Icon.png"
```

Comment:
This should use pattern matching (or some other Haskell trick; can this be improved @balacij?)
```suggestion
drasilImage "Projectile" = alignImage "../../../../../drasil-website/WebInfo/images/Icon.png"
drasilImage _            = alignImage "../../../../drasil-website/WebInfo/images/Icon.png"
```

Reply:
I'm really uncomfortable with hard-coding any of our examples into the rest of Drasil. 

Much better would be some kind of other way to signal to the Markdown printer (from the example) the relative location. For example, maybe `drasilImage` could use some kind of (to be created) `buildPath` function which would take a natural number corresponding to the number of `..` to add. Such "configuration data" could then all be put into a record that could be passed around.

Reply:
I like that as well! I will look into creating the `buildPath` function now.